backend/src/test/java/de/mcstangl/projectplanner/service/UserServiceTest.java

```diff
@@ -0,0 +1,103 @@
+package de.mcstangl.projectplanner.service;
+
+import de.mcstangl.projectplanner.model.UserEntity;
+import de.mcstangl.projectplanner.repository.UserRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+class UserServiceTest {
+
+    @Mock
+    private UserRepository userRepositoryMock;
+
+    @InjectMocks
+    private UserService userService;
+
+
+    private AutoCloseable closeable;
+
+
+    @BeforeEach
+    void setup() {
+        closeable = MockitoAnnotations.openMocks(this);
+
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+        closeable.close();
+    }
+
+    @Test
+    @DisplayName("Find all shoul return all user in DB")
```

Comment:
```suggestion
    @DisplayName("Find all should return all user in DB")
```

Wenn ich sonst schon nix finde, dann wenigstens das :D

Reply:
danke  :)
👍